nimble/host/mesh/syscfg.yml

```diff
@@ -191,6 +191,12 @@ syscfg.defs:
             as there are outgoing segments.
         value: 3
 
+    BLE_MESH_SEG_RETRANSMIT_ATTEMPTS:
+        description: >
+            Number of retransmit attempts (after the initial transmit) per segment
```

Comment:
Please put here specification reference we discussed, so it is clear where restriction came from.

Reply:
@rymanluk Sure I will put the specification reference. Also, the restriction doesn't work if I implement the way I have done. I have tried $notnull but I didn't get any error message. Could you please let me know the better way to implement the restrictions. Thank you!